resources/fileTemplates/internal/Magento Routes XML.xml.html

```diff
@@ -30,14 +30,43 @@
     <tr>
         <td valign="top"><nobr><font face="verdana" size="-2"><b>${ROUTER_ID}</b></font></nobr></td>
         <td width="10">&nbsp;</td>
-        <td width="100%" valign="top"><font face="verdana" size="-1">specifies the name of the router in Magento.
+        <td width="100%" valign="top"><font face="verdana" size="-1">Specifies the name of the router in Magento.
             See the reference tables in the
             <a href="https://devdocs.magento.com/guides/v2.4/extension-dev-guide/routing.html#router-class">
                 Router class section
             </a>.
         </font>
         </td>
     </tr>
+    <tr>
+        <td valign="top"><nobr><font face="verdana" size="-2"><b>${HAS_ROUTE_DATA}</b></font></nobr></td>
+        <td width="10">&nbsp;</td>
+        <td width="100%" valign="top"><font face="verdana" size="-1">Technical value used to enable/disable route information rendering
+        </font>
+        </td>
+    </tr>
+    <tr>
+        <td valign="top"><nobr><font face="verdana" size="-2"><b>${ROUTE_ID}</b></font></nobr></td>
+        <td width="10">&nbsp;</td>Specifies the unique node id for this route in Magento,
+        is also the first segment of its associated layout handle XML filename: routeId_controller_action.xml
+        <td width="100%" valign="top"><font face="verdana" size="-1">
+        </font>
+        </td>
+    </tr>
+    <tr>
+        <td valign="top"><nobr><font face="verdana" size="-2"><b>${FRONT_NAME}</b></font></nobr></td>
+        <td width="10">&nbsp;</td>
+        <td width="100%" valign="top"><font face="verdana" size="-1">Specifies the first segment after the base URL of a request
+        </font>
+        </td>
+    </tr>
+    <tr>
+        <td valign="top"><nobr><font face="verdana" size="-2"><b>${MODULE_NAME}</b></font></nobr></td>
+        <td width="10">&nbsp;</td>
+        <td width="100%" valign="top"><font face="verdana" size="-1">Specifies the name of your module
+        </font>
+        </td>
+    </tr>
 </table>
 </body>
 </html>
```

Comment:
Could you please add new line at end of file

Reply:
Done